specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/playbookConfigurations.json

```diff
@@ -0,0 +1,461 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2019-01-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [ "https" ],
+  "consumes": [ "application/json" ],
+  "produces": [ "application/json" ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/playbookConfigurations": {
+      "get": {
+        "x-ms-examples": {
+          "Get playbook configurations of subscription": {
+            "$ref": "./examples/PlaybookConfigurations/GetPlaybookConfigurationsSubscription_example.json"
+          }
+        },
+        "tags": [ "PlaybookConfigurations" ],
+        "description": "Get all playbook configurations in a subscription.",
+        "operationId": "PlaybookConfigurations_ListBySubscription",
```

Comment:
please change to "PlaybookConfigurations_List"

Reply:
This name is already taken by another operationid (List resources under a given resource group)